src/univariate/continuous/ksdist.jl

```diff
@@ -73,12 +73,12 @@ function cdf_durbin(d::KSDist,x::Float64)
     k, ch, h = ceil_rems_mult(n,x)
 
     m = 2*k-1
-    H = Array(Float64,m,m)    
+    H = Array(Float64,m,m)
     for i = 1:m, j = 1:m
-        H[i,j] = i-j+1 >= 0 ? 1.0 : 0.0        
+        H[i,j] = i-j+1 >= 0 ? 1 : 0
     end
-    r = 1.0
-    for i = 1:m 
+    r = 1
```

Comment:
We didn't parameterize any distribution with only integer arguments. One could perhaps want an Int32 here, but the original motivation, automatic differentiation, wouldn't apply for these cases with only discrete parameters.